open-metadata-test/open-metadata-fvt/access-services-fvt/governance-program-fvt/pom.xml

```diff
@@ -73,6 +73,12 @@
             <artifactId>governance-leadership-sample</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
harmless, but we *should* be ok here as long as the assembly (which I think is the aim of what we test) is dependent on the sample.

Where is the actual code that executes the tests (I may have missed in lots of files)? Ideally I think that would be invoked via gradle or in pom via surefire plugin, similar to how we launch the other fvts?

Reply:
Ah. I see the test is in https://github.com/mandy-chessell/egeria/commit/c6dfbf297b838a33ff3358f2074cf7415e8bc6eb